Documentation/minio-object-store-crd.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Minio Object Store CRD
+weight: 40
+indent: true
+---
+
+# Minio Object Store CRD
+Minio object stores can be created and configured using the `objectstores.minio.rook.io` custom resource definition (CRD). Complete instructions can be found in the [Rook Minio Documentation](minio-object-store.md).
+
+##Sample
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.minio.rook.io
+spec:
+  group: minio.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
+  scope: Namespaced
+  version: v1alpha1
+```
+
+## Cluster Settings
+
+### Minio Specific Settings
+
+The settings below are specific to Minio object stores:
+
+* `credentials`: This accepts the `name` and `namespace` of an existing Secret to specify the access credentials for the object store.
```

Comment:
maybe a few more details about the expected format of the secret, but that could be more clear once the sample above has the CRD instance instead of the CRD type as content.